Library/Homebrew/manpages.rb

```diff
@@ -31,8 +35,15 @@ def self.regenerate_man_pages(quiet:)
       Homebrew.install_bundler_gems!(groups: ["man"])
 
       markup = build_man_page(quiet:)
-      convert_man_page(markup, TARGET_DOC_PATH/"Manpage.md")
-      convert_man_page(markup, TARGET_MAN_PATH/"brew.1")
+      root, = Parser::Ronn.parse(markup)
```

Comment:
Also output the warnings here?

Reply:
Currently spams `No link definition for link ID` because of all the `[...]` we have in command banners even though it does successfully figure out that it's not a link. I guess I could try `gsub!(/[[]]/, "\\\\0")` those.